core/entity/data/v1/v1.go

```diff
@@ -70,10 +70,11 @@ func (t *JsonTime) UnmarshalJSON(data []byte) error {
 }
 
 var (
-	Memory BackingStore = "memory"
-	File   BackingStore = "file"
-	Json   SecretFormat = "json"
-	Yaml   SecretFormat = "yaml"
+	Memory   BackingStore = "memory"
+	File     BackingStore = "file"
+	Json     SecretFormat = "json"
+	Yaml     SecretFormat = "yaml"
+	FreeForm SecretFormat = "freeForm"
```

Comment:
I like the idea.

Can we rename it as `"raw"` instead?

`Raw SecretFormat = "raw"` ?

It will imploy that the secret is in its original state without any specific formatting or structure applied.